youtube_dl/extractor/polskieradio.py

```diff
@@ -87,6 +90,7 @@ def _real_extract(self, url):
                 'duration': int_or_none(media.get('length')),
                 'vcodec': 'none' if media.get('provider') == 'audio' else None,
                 'timestamp': timestamp,
+                'thumbnail': thumbnail_url
```

Comment:
Just pass in `self._og_search_thumbnail()` directly here, since it is not used anywhere else